expression/builtin_string_vec_test.go

```diff
@@ -35,6 +35,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Upper: {
 		{types.ETString, []types.EvalType{types.ETString}, nil},
 	},
+	ast.Space: {
+		{types.ETString, []types.EvalType{types.ETInt}, []dataGenerator{&rangeInt64Gener{-10, 2000}}},
```

Comment:
How about adding one more test which generates strings with length around 8 since the estimated length of strings in `ReserveString` is 8.

Reply:
done, PTAL